RadixWallet/Clients/ROLAClient/ROLAClient+Live.swift

```diff
@@ -144,34 +145,8 @@ extension ROLAClient {
 	}
 }
 
-extension EngineToolkit.PublicKeyHash {
-	public struct InvalidPublicKeyHashLength: Error {
-		public let got: Int
-		public let expected: Int
-	}
-
-	static let hashLength = 29
-
-	public init(hashing publicKey: SLIP10.PublicKey) throws {
-		let hashBytes = try blake2b(data: publicKey.compressedData).suffix(Self.hashLength)
-
-		guard
-			hashBytes.count == Self.hashLength
-		else {
-			throw InvalidPublicKeyHashLength(got: hashBytes.count, expected: Self.hashLength)
-		}
-
-		switch publicKey {
-		case .ecdsaSecp256k1:
-			self = .secp256k1(value: hashBytes)
-		case .eddsaEd25519:
-			self = .ed25519(value: hashBytes)
-		}
-	}
-}
-
 extension OnLedgerEntity.Metadata {
-	public func ownerKeyHashes() throws -> [EngineToolkit.PublicKeyHash]? {
+	public func ownerKeyHashes() throws -> [RETPublicKeyHash]? {
```

Comment:
Shall drop the `RET` prefix... but we a naming collision with both `OnLedgerEntity.Metadata` and Gateway client models.